src/main.jsx

```diff
@@ -62,6 +59,8 @@ let App = React.createClass({
   render() {
     return <div className="main">
       <RouteHandler current={this.state.current}
+                    user={this.state.current.user}
```

Comment:
Nu. Vreau sa fac `getDefaultProps` in caz ca nu imi seteaza parintele chestiile astea si nu va face deep merge la current. O sa trimitem tot ce vine din current ca single nested object ca sa putem baga defaults.